src/SFA.DAS.ContentApi.Database/Tables/Client.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Client]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [Description] VARCHAR(50) NULL, 
+    [Identity] NVARCHAR(20) NOT NULL, 
```

Comment:
Why is this using a NVARCHAR - what is stored here - it seems more likely that the description would contain unicode characters than an identifier?